### PR TITLE
Update PPP - Song Yi

### PR DESCRIPTION
**Pull request by songyiang**

### CODE REVIEW

File: docs/team/songyi98.md [hunk 1]
```diff
@@ -5,45 +5,43 @@ title: Song Yi's Project Portfolio Page
...
 Users interact with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
 
 This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org/).
 
 * **New Feature**: Added the expiry date attribute.
   * What it does: The expiry date attribute allows the user to add an expiry date to a food item.
   * Justification: This feature is a core part of our food inventory management. It allows us to keep track of which food item is expiring or has expired.
-  * Highlights: Parsing of expiry date to ensure validity is challenging as there is no single formatter in Java that can capture all formatting and invalid date errors. In the end, SimpleDateFormat and DateTimeFormatter are used.
+  * Highlights: Parsing of expiry date to ensure validity is challenging as there is no single formatter in Java that can capture all formatting and invalid date errors. In the end, `SimpleDateFormat` and `DateTimeFormatter` are used.
```
Comment (bchenghi) [inline docs/team/songyi98.md]: Should it be "were used" at the end of sentence? Since it starts with "In the end".

File: docs/team/songyi98.md [hunk 2]
```diff
@@ -5,45 +5,43 @@ title: Song Yi's Project Portfolio Page
...
 
 * **New Feature**: Added the change quantity command.
   * What it does: It changes the quantity of a food item by a certain amount as entered by the user.
   * Justification: This helps to relieve the burden of having to calculate the final amount of food item the user will have, making adding and subtracting food quantity much easier.
   * Highlights:
-    * Parsing of amount is challenging as there are numerous issues when dealing with double arithmetics such as unprecise decimal placing, rounding errors and numeric overflow.
+    * Parsing of amount is challenging as there are numerous issues when dealing with double arithmetics such as imprecise decimal placing, rounding errors and numeric overflow.
     * The regex for accepting a signed number with a maximum of 2 decimal places requires a lot of fine-tuning to get it right.
-    * Methods have to be created in the Quantity class to prevent breaking the Law of Demeter while changing the quantity from the ChangeQuantityCommand class.
+    * Methods have to be created in the Quantity class to prevent breaking the Law of Demeter while changing the quantity of a food item from the ChangeQuantityCommand class.
```
Comment (bchenghi) [inline docs/team/songyi98.md]: Should this be "had to be created" instead?

File: docs/team/songyi98.md [hunk 3]
```diff
@@ -5,45 +5,43 @@ title: Song Yi's Project Portfolio Page
...
+    * Added documentation and implementation details of the `changeqty` feature (e.g. [\#99](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/99), [\#191](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/191))
 
-* **Tools**:
-  * Integrated a third party library (Natty) to the project ([\#42]())
-  * Integrated a new Github plugin (CircleCI) to the team repo
+* **Project management and contributions to team-based tasks**:
+  * Contributed to planning and ideation of application
+  * Setting up the GitHub team org and repo
+  * Enabled assertions in the gradle file (e.g. [\#82](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/82))
+  * Standardised formatting of User Guide, update table of contents with hyperlinks and update About this Document section (e.g. [\#127](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/127))
```
Comment (bchenghi) [inline docs/team/songyi98.md]: Should it be "updated" instead?

File: docs/team/songyi98.md [hunk 4]
```diff
@@ -5,45 +5,43 @@ title: Song Yi's Project Portfolio Page
...
-    * Added documentation and implementation details of the `changeqty` feature: [\#99](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/99) and []() **TBU**
+    * Added use case and user stories (e.g. [\#80](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/80))
+    * Added documentation and implementation details of the `changeqty` feature (e.g. [\#99](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/99), [\#191](https://github.com/AY2021S1-CS2103T-F13-4/tp/pull/191))
 
-* **Tools**:
-  * Integrated a third party library (Natty) to the project ([\#42]())
-  * Integrated a new Github plugin (CircleCI) to the team repo
+* **Project management and contributions to team-based tasks**:
+  * Contributed to planning and ideation of application
+  * Setting up the GitHub team org and repo
```
Comment (bchenghi) [inline docs/team/songyi98.md]: Should this be "Set up" instead?

**Review by bchenghi — APPROVED** [review]
LGTM. Just some minor comments.

File: docs/team/songyi98.md (hunk 1 above)
Comment (songyiang) [reply docs/team/songyi98.md]: Changed. Thanks.

File: docs/team/songyi98.md (hunk 2 above)
Comment (songyiang) [reply docs/team/songyi98.md]: Changed. Thanks.

File: docs/team/songyi98.md (hunk 3 above)
Comment (songyiang) [reply docs/team/songyi98.md]: Changed. Thanks.

File: docs/team/songyi98.md (hunk 4 above)
Comment (SONGYI98) [reply docs/team/songyi98.md]: Changed. Thanks.

Comment (songyiang) [reply docs/team/songyi98.md]: Changed. Thanks.

**Pull request merged**